core/src/main/java/org/elasticsearch/index/engine/Engine.java

```diff
@@ -486,10 +488,10 @@ protected final GetResult getFromSearcher(Get get, Function<String, Searcher> se
     }
 
     public final GetResult get(Get get) throws EngineException {
```

Comment:
If we totally delete this, we also need to replace the call of this method in `InternalEngineTests` and `RefreshListenersTests` by:
`get(Get get, Function<String, Searcher> searcherFactory, LongConsumer onRefresh)`
ex.https://github.com/PnPie/elasticsearch/blob/test/core/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java#L923 
For non-realtime get we can pass `null` for `onRefresh` consumer, for realtime get so we just pass sth. whatever like `(x) -> {}` ?